modules/bullet/space_bullet.h

```diff
@@ -110,17 +110,23 @@ class SpaceBullet : public RIDBullet {
 	real_t linear_damp = 0.0;
 	real_t angular_damp = 0.0;
 
-	Vector<CollisionObjectBullet *> collision_objects;
-	Vector<AreaBullet *> areas;
+	LocalVector<CollisionObjectBullet *> queue_pre_flush;
+	LocalVector<CollisionObjectBullet *> queue_flush;
+	LocalVector<CollisionObjectBullet *> collision_objects;
+	LocalVector<AreaBullet *> areas;
 
-	Vector<Vector3> contactDebug;
-	int contactDebugCount = 0;
+	LocalVector<Vector3> contactDebug;
+	uint32_t contactDebugCount = 0;
```

Comment:
Shouldn't these be `snake_case`?

Reply:
Yep, it should be. Also I would check if make sense remove it (now that we have `LocalVector`)